metrics/platform-metrics-webconsole-plugin/pom.xml

```diff
@@ -97,6 +97,7 @@
                         <Import-Package>
                             javax.servlet;version="[2.5,3)",
                             javax.servlet.http;version="[2.5,3)",
+                            org.apache.felix.webconsole; version="[3.1.0,4.0.0)",
```

Comment:
Yes. With 2.4.3 the exported version was 3.1.2 but this was looking for the range [3.2.0,4.0.0). Will this be an issue?